src/modules/vendor/components/programs/cards/templates/MultipleChoiceQuestion.vue

```diff
@@ -86,6 +86,7 @@ export default {
     },
     async updateAnswer (index) {
       try {
+        this.card.qcmAnswers[index].label = this.card.qcmAnswers[index].label.trim();
         if (this.tmpInput === get(this.card, `answers[${index}].label`)) return;
```

Comment:
ici ca ne devrait pas plutot etre `get(this.card, qcmAnswers[${index}].label))` ?

Reply:
C'est très probable, puisque le template n'a besoin que la variable qcmAnswers et pas de la variable answers.
Tu peux tester pour t'en rendre compte : si tu focus l'input sans changer la réponse puis tu sors du focus de l'input, tu devrais avoir la notification 'Carte mise à jour.', ce test étant là pour ne rien renvoyer si ce qu'il y a dans l'input est déjà égal à la réponse enregistrée

Reply:
yes merci pour lexplic :)